src/gen_rpc_acceptor.erl

```diff
@@ -197,7 +197,10 @@ call_worker(Parent, WorkerPid, Ref, M, F, A) ->
     % and manifest as timeout. Wrap inside anonymous function with catch
     % will crash the worker quickly not manifest as a timeout.
     % See call_MFA_undef test.
-    % Fun = fun() -> catch erlang:apply(M, F, A) end,
-    PacketBin = erlang:term_to_binary({WorkerPid, Ref, catch erlang:apply(M, F, A)}),
+    Ret = case catch erlang:apply(M, F, A)of
```

Comment:
This part is a direct rip off from the rpc src code :D.
They put {badrpc, {'EXIT, Stuff}} to match rpc(someothernode, F, A).